server/chat-plugins/mafia.ts

```diff
@@ -1,6 +1,7 @@
 import {Utils, FS} from '../../lib';
 
 interface MafiaData {
+	[key: string]: any;
```

Comment:
This typing says "you can put anything in a MafiaData object at all! We have no idea whats in here as a result" and thats bad. Is there a more specific typing you could be using? Why do we need this typing?